lib/CL/devices/level0/pocl-level0.cc

```diff
@@ -1064,8 +1064,11 @@ int pocl_level0_alloc_mem_obj(cl_device_id ClDevice, cl_mem Mem, void *HostPtr)
     P->mem_ptr = Mem->mem_host_ptr;
     P->version = Mem->mem_host_ptr_version;
   } else {
-    //    void *all = Device->allocDeviceMem(mem->size);
-    Allocation = Device->allocSharedMem(Mem->size);
+    bool Compress = false;
+    if (pocl_get_bool_option("POCL_LEVEL0_COMPRESS", 0)) {
```

Comment:
This env should be more descriptive (POCL_LEVEL0_COMPRESS_RO_BUFFERS or similar) and documented. Also, is there a reason not to use this in case compression is available? It's best to avoid extra code paths if possible.

Reply:
I don't want it documented (yet), this is an experimental flag. I want to measure if it has any performance impact at all (so far i haven't seen any, but haven't done much testing). But good point about the code paths, i'll just enable the compression if it proves useful and remove the variable.